homeassistant/components/citybikes/sensor.py

```diff
@@ -239,4 +240,5 @@ async def async_update(self) -> None:
             ATTR_LONGITUDE: station.longitude,
             ATTR_EMPTY_SLOTS: station.empty_slots,
             ATTR_TIMESTAMP: station.timestamp,
+            ATTR_EBIKE: station.extra.get(ATTR_EBIKE),
```

Comment:
What information does this contain?

../Frenck

Reply:
The thing is, we are slowly adding more and more information to this location. I almost wonder if the location should be a "device" in home assistant, with separate entities for information like this.

Reply:
Updated the PR to also have entities for the device.

<img width="1121" height="677" alt="Screenshot 2025-11-11 at 11 41 43 PM" src="https://github.com/user-attachments/assets/4833db09-d765-4655-baa8-cb355280f9c1" />
<img width="1129" height="835" alt="Screenshot 2025-11-11 at 11 42 12 PM" src="https://github.com/user-attachments/assets/0f7d5c73-d3a4-47dc-88a6-47d7b0226b03" />